functions/exercise5/scriptSpect.js

```diff
@@ -0,0 +1,20 @@
+/// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals convertWordFirstLetterInUpperCase*/
+describe("Converted the first letter of word of the string in upper case:", function () {
```

Comment:
"Converting the first letter of each word in the string into upper case" sounds better